test/src/module/common/ioHttpTest.c

```diff
@@ -626,6 +626,18 @@ testRun(void)
                 hrnServerScriptClose(http);
                 hrnServerScriptAccept(http);
 
+                hrnServerScriptExpectZ(http, "GET / HTTP/1.1\r\n" TEST_USER_AGENT "\r\n");
+                hrnServerScriptReplyZ(http, "HTTP/1.1 408 Request Timeout\r\ncontent-length:0\r\nConnection:close\r\n\r\n");
+
+                hrnServerScriptClose(http);
+                hrnServerScriptAccept(http);
+
+                hrnServerScriptExpectZ(http, "GET / HTTP/1.1\r\n" TEST_USER_AGENT "\r\n");
+                hrnServerScriptReplyZ(http, "HTTP/1.1 429 To Many Requests\r\ncontent-length:0\r\nConnection:close\r\n\r\n");
```

Comment:
Probably doesn't matter, but should it be `Too Many Requests`?  Assuming the message is mainly human-readable.

Reply:
It doesn't hurt anything because it is a test but it is always better to be tidy. Fixed in c4a481406b1.